### PR TITLE
Feature/234  implement degree object

### PR DESCRIPTION
**Pull request by eclemmer**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/objects/Education_History__c/fields/Status__c.field-meta.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Status__c</fullName>
+    <description>The status of the Education History. For example, In Progress or Completed.</description>
```
Comment (MatthewBlanski) [inline force-app/main/default/objects/Education_History__c/fields/Status__c.field-meta.xml:4]: We don't have an In Progress picklist value. I've reached out to docs on this one to get a final text example here, but we will need to change this.

File: force-app/main/default/layouts/Program_Enrollment__c-HEDA Program Enrollment Layout.layout-meta.xml [hunk 2]
```diff
@@ -14,62 +14,88 @@
...
+                <field>Start_Date__c</field>
             </layoutItems>
             <layoutItems>
                 <behavior>Edit</behavior>
-                <field>Admission_Date__c</field>
+                <field>Class_Standing__c</field>
             </layoutItems>
+        </layoutColumns>
+        <layoutColumns>
             <layoutItems>
```
Comment (MatthewBlanski) [inline force-app/main/default/layouts/Program_Enrollment__c-HEDA Program Enrollment Layout.layout-meta.xml:31]: This is missing a layoutItem for a blank space outlined in the screenshot for implementation, changing the tab order of fields.

File: force-app/main/default/flexipages/EDA_Academic_Certification_Record_Page.flexipage-meta.xml [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                </componentInstanceProperties>
+                <componentInstanceProperties>
+                    <name>hideChatterActions</name>
+                    <value>false</value>
+                </componentInstanceProperties>
+                <componentInstanceProperties>
+                    <name>numVisibleActions</name>
+                    <value>3</value>
+                </componentInstanceProperties>
+                <componentName>force:highlightsPanel</componentName>
```
Comment (MatthewBlanski) [inline force-app/main/default/flexipages/EDA_Academic_Certification_Record_Page.flexipage-meta.xml:26]: The highlights panel is not visible in the default version of this and is not pulling up the requested fields in the document.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Some minor changes requested on this item, largely resolving around small bits of metadata.

File: force-app/main/default/objects/Education_History__c/fields/Status__c.field-meta.xml (hunk 1 above)
Comment (MatthewBlanski) [reply force-app/main/default/objects/Education_History__c/fields/Status__c.field-meta.xml:4]: Update on this. We should change `In Progress` to `Enrolled`

Comment (eclemmer) [reply force-app/main/default/objects/Education_History__c/fields/Status__c.field-meta.xml:4]: Fixed

File: force-app/main/default/layouts/Program_Enrollment__c-HEDA Program Enrollment Layout.layout-meta.xml (hunk 2 above)
Comment (eclemmer) [reply force-app/main/default/layouts/Program_Enrollment__c-HEDA Program Enrollment Layout.layout-meta.xml:31]: Fixed

File: force-app/main/default/flexipages/EDA_Academic_Certification_Record_Page.flexipage-meta.xml (hunk 3 above)
Comment (eclemmer) [reply force-app/main/default/flexipages/EDA_Academic_Certification_Record_Page.flexipage-meta.xml:26]: Fixed

File: unpackaged/config/trial/profiles/Admin.profile [hunk 4]
```diff
@@ -121,9 +121,6 @@
         <layout>%%%NAMESPACE%%%Course_Enrollment__c-%%%NAMESPACE%%%HEDA Course Enrollment Layout</layout>
         <recordType>%%%NAMESPACE%%%Course_Enrollment__c.Student</recordType>
     </layoutAssignments>
-    <layoutAssignments>
-        <layout>Opportunity-%%%NAMESPACE%%%EDA Opportunity Layout</layout>
-    </layoutAssignments>
```
Comment (MatthewBlanski) [inline unpackaged/config/trial/profiles/Admin.profile:126]: Please revert this change.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
Everything checks out, but one of the other changes from another PR got caught up in the mix. Please restore that to the admin profile for the trial org flow.

**Review by MatthewBlanski — APPROVED** [review]
